docs/sources/mimir/configure/configuration-parameters/index.md

```diff
@@ -2669,6 +2674,15 @@ alertmanager_client:
   # CLI flag: -alertmanager.alertmanager-client.connect-backoff-max-delay
   [connect_backoff_max_delay: <duration> | default = 5s]
 
+  cluster_validation:
+    # (experimental) Optionally define client's cluster validation label.
+    # CLI flag: -alertmanager.alertmanager-client.cluster-validation.label
+    [label: <string> | default = ""]
+
+  # (experimental) Optionally define gRPC client's cluster validation label.
```

Comment:
Same, is there more than one gRPC client? Is yes, we could say `a gRPC client`. If no, we could say `the gRPC client`.